<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DSHYD-106" name="com.RM.scripts.TC6_ExpiryCardError" tests="2" failures="0" timestamp="15 Nov 2017 10:54:29 GMT" time="127.197" errors="1">
  <testcase name="tearDownFirefox" time="33.324" classname="com.RM.scripts.TC6_ExpiryCardError">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0
	at java.util.Collections$EmptyList.get(Collections.java:4454)
	at java.util.Collections$UnmodifiableList.get(Collections.java:1309)
	at com.RM.accelerators.TestEngine.setElements(TestEngine.java:734)
	at com.RM.accelerators.TestEngine.WriteExcelfile(TestEngine.java:667)
	at com.RM.accelerators.TestEngine.tearDownFirefox(TestEngine.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:350)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- tearDownFirefox -->
  <testcase name="testTC6_ExpiryCardError" time="93.873" classname="com.RM.scripts.TC6_ExpiryCardError"/>
</testsuite> <!-- com.RM.scripts.TC6_ExpiryCardError -->
